### DISCUSSION TITLE
Report or upload missing (hiking) routes

### DISCUSSION THREAD

**Discussion by HeinPauwelyn** [waymarkedtrails/waymarked-trails-site#436] [Q&A]
I've seen that there is a missing hinking route in the region of Damme (nearby Bruges, Flanders, Belgium). Where could I report or upload it?

- Link to the information page of the roure: https://www.visitwestvlaanderen.be/nl/damse-vaartwandelroute

**Comment by MFCON** [comment]
You need to enter the data on openstreetmap.org, where the data is all held. Hiking routes are all tagged as "relations"
